assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -203,12 +217,20 @@ function FontPickerContainer({ value, onSelect, onClose }) {
   return (
     <PickerContainer ref={ref}>
       <SearchInput
-        type="text"
+        type="search"
+        role="combobox"
+        aria-expanded={Boolean(matchingFonts.length)}
+        aria-autocomplete="list"
+        aria-owns="editor-font-picker-list"
         value={searchKeyword}
+        onKeyDown={handleKeyPress}
+        onFocus={handleInputFocus}
+        placeholder={__('Search', 'web-stories')}
         onChange={handleSearchInputChanged}
       />
       {matchingFonts.length ? (
         <List
+          id="editor-font-picker-list"
           onKeyDown={handleKeyPress}
           items={matchingFonts}
           onScroll={handleScroll}
```

Comment:
This is not invoked when the contents change (as contents never could change before), but that means that for a given small search result where no scroll ever happens, the custom font faces aren't loaded and thus just displayed with the default font:

![Screen Shot 2020-06-09 at 10 01 41 PM](https://user-images.githubusercontent.com/637548/84218824-31c80180-aa9d-11ea-9640-2a55fe68c7af.png)

Reply:
Also, what's with the red border?

Reply:
I’m not sure. I’ve never seen that. Perhaps a bowser style. So the issue with scroll is we set up the panel to dismiss any drop downs when there is a scroll event. It’s why the font drop down used to immediately close when it was opened. I think we have competing UX here.